base/generator.jl

```diff
@@ -0,0 +1,21 @@
+"""
+    Generator(f, iter)
+
+Given a function `f` and an iterator `iter`, construct an iterator that yields
+the values of `f` applied to the elements of `iter`.
+The syntax `f(x) for x in iter` is syntax for constructing an instance of this
+type.
+"""
```

Comment:
i think you need to add this signature to the appropriate manual section and then run genstdlib.jl to copy this text over